src/main/java/io/ebeaninternal/dbmigration/migrationreader/MigrationXmlReader.java

```diff
@@ -40,22 +44,22 @@ public static Migration read(File migrationFile) {
     try (FileInputStream is = new FileInputStream(migrationFile)) {
       return read(is);
     } catch (IOException e) {
-      throw new RuntimeException(e);
+      throw new RuntimeException("Could not read resource: " + migrationFile, e);
     }
   }
 
   /**
    * Read and return a Migration from an xml document.
    */
-  public static Migration read(InputStream is) {
+  public static Migration read(InputStream is) throws IOException {
 
     try {
       JAXBContext jaxbContext = JAXBContext.newInstance(Migration.class);
       Unmarshaller unmarshaller = jaxbContext.createUnmarshaller();
       return (Migration) unmarshaller.unmarshal(is);
 
     } catch (JAXBException e) {
-      throw new RuntimeException(e);
+      throw new IOException(e);
```

Comment:
if read will fail, the excepton is wrapped in an IoExcecption, this exception is catched by the caller and wrapped in a RuntimeException with the resource name.
So, this change should ensure, that the filename of the exception is in the stack trace.